templates/wizards/projects/asn1acn/file.pro

```diff
@@ -0,0 +1,13 @@
+TEMPLATE = app
+CONFIG += console
+CONFIG -= app_bundle
+CONFIG -= qt
+
+%{JS: if ('%{AddAsnFile}' === 'true' && '%{AddAcnFile}' === 'true')
+        'DISTFILES += \\\\ \n %{AsnFullFileName} \\\\ \n %{AcnFullFileName}'
```

Comment:
in future we'll probably want to use something different than `DISTFILES` (aren't those files copied with distribution? need checking)